pkgs/tools/misc/coreutils/default.nix

```diff
@@ -22,22 +22,13 @@ with lib;
 
 stdenv.mkDerivation (rec {
   pname = "coreutils${optionalString (!minimal) "-full"}";
-  version = "9.0";
+  version = "9.1";
 
   src = fetchurl {
     url = "mirror://gnu/coreutils/coreutils-${version}.tar.xz";
-    sha256 = "sha256-zjCs30pBvFuzDdlV6eqnX6IWtOPesIiJ7TJDPHs7l84=";
+    sha256 = "sha256:08q4b0w7mwfxbqjs712l6wrwl2ijs7k50kssgbryg9wbsw8g98b1";
```

Comment:
This branch has gotten all weird.  9.0 -> 9.1 instead of 9.3 -> 9.4 &ndash; and the hash format.